tests/distributed/test_distributed_sampling.py

```diff
@@ -1858,6 +1859,92 @@ def test_local_sampling_heterograph(num_parts, use_graphbolt, prob_or_mask):
                 )
 
 
+def check_mask_hetero_sampling_gb(tmpdir, num_server, use_graphbolt=True):
+    def create_hetero_graph(dense=False, empty=False):
```

Comment:
I see there's already a function named `create_random_hetero()` could that be used here?

Reply:
The graph created by create_random_hetero() contains r12, r23, and r13 relations.

When applying a mask to r12, neither DGLGraph nor GB format can sample the n3 node.
When applying a mask to r23, both DGLGraph and GB format can sample the n1 node.
When applying a mask to r13, both DGLGraph and GB format can sample the n2 node.

So the graph can't be used here and I create a new function.

Reply:
OK. Can we add a comment why this particular graph structure is needed for this test?

Reply:
Oh, I re-ran the tests and found that the previous statement was incorrect regarding the use of the `create_random_hetero`. Specifically, before the bug fix, `n1` couldn't be sampled when mask was only applied to `r23`, but after the fix, `n1` can now be sampled by `r13`. So, I've switched to using `create_random_hetero` for the tests.